src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/Huffman.cs

```diff
@@ -406,13 +406,13 @@ private static int DecodeValue(uint data, int validBits, out int decodedBits)
 
             int codeMax = 0;
 
-            for (int i = 0; i < s_decodingTable.Length && s_decodingTable[i].codeLength <= validBits; i++)
+            for (int i = 0; i < _decodingTable.Length && _decodingTable[i].codeLength <= validBits; i++)
```

Comment:
Another example where I'm led astray by the `_`.  If this were actually an instance field, perf would be better if it were instead first stored into a local.